llama-index-core/llama_index/core/readers/file/base.py

```diff
@@ -10,6 +10,7 @@
 from itertools import repeat
 from pathlib import Path
 import fsspec
+from gcsfs import GCSFileSystem
```

Comment:
I think adding this dependency here adds a hard dependency on gcsfs? We probably don't want to do that